osgi/pom.xml

```diff
@@ -104,7 +103,8 @@
                             commons-math3,
                             commons-compress,
                             commons-collections4,
-                            commons-codec
+                            commons-codec,
+                            log4j-api
```

Comment:
I don't know how this works or how to verify it. I'd appreciate any help or guidance.

Reply:
remove any unnecessary slf4j or commons-logging entries